supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -8,6 +8,16 @@
 
 *See also*: xref:paas[PaaS], xref:saas[SaaS]
 
+[[ibm-cloud-bare-metal]]
+==== image:images/yes.png[yes] IBM Cloud® Bare Metal (Classic)
+*Description*: _IBM Cloud® Bare Metal (Classic)_ represents a platform where a customer can run an OpenShift Container Platform cluster on an IBM bare-metal server. Ensure that "IBM Cloud®" precedes "Bare Metal (Classic)". On first use, mark "IBM Cloud" Bare Metal (Classic) with a registered trademark icon (®).
```

Comment:
```suggestion
*Description*: _IBM Cloud® Bare Metal (Classic)_ refers to a platform where a customer can run an OpenShift Container Platform cluster on an IBM bare-metal server. Ensure that "IBM Cloud®" precedes "Bare Metal (Classic)". On first use, mark "IBM Cloud" Bare Metal (Classic) with a registered trademark icon (®).
```
Nitpick: Maybe "refers to" is a bit more accurate than "represents" here?

Reply:
Thank you. A worthy nit :). Updated!